etc/thunderbird-beta.profile

```diff
@@ -0,0 +1,35 @@
+# Firejail profile for thunderbird
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/thunderbird.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+# Users have thunderbird set to open a browser by clicking a link in an email
+# We are not allowed to blacklist browser-specific directories
+whitelist /opt/thunderbird-beta
+noblacklist ${HOME}/.cache/thunderbird
```

Comment:
Everything below this line can be replaced with the two lines
```
# Redirect
include /etc/firejail/thunderbird.profile
```
This will redirect firejail to use thunderbird's profile while still whitelisting /opt/thunderbird-beta.